### ISSUE TITLE
Can you provide an example how to use it using `stateful metrics`?

### ISSUE THREAD

**Issue by sudonto** [yueyericardo/pkbar#3]
Nice code, thanks. Could you provide an example how to display metrics as it is (no average)? Thanks

**Comment by yueyericardo** [comment]
Hi,

The Kbar's code is taken directly from tf.keras.utils.Progbar.
And it does not support stateful metrics.

I will try to add this feature when I find some time.

**Comment by sudonto** [comment]
After reading your code, I understand how to use `stateful_metrics`. Just pass variables you dont want to take the average to param `stateful_metrics` and you will get values as it is (no avg). This is my example:
Declaration:
`kbar = pkbar.Kbar(target=len(data_loader), width=8,
                      stateful_metrics=['loss','top1','top5'])`

Update progbar:
`kbar.update(i, values=[('batch_time', batch_time.val),
                               ('loss', losses.avg),
                               ('top1', top1s.avg), 
                               ('top5', top5s.avg)])`
 
I like your work, thanks!

**Comment by yueyericardo** [comment]
> After reading your code, I understand how to use `stateful_metrics`. Just pass variables you dont want to take the average to param `stateful_metrics` and you will get values as it is (no avg). This is my example:
> Declaration:
> `kbar = pkbar.Kbar(target=len(data_loader), width=8, stateful_metrics=['loss','top1','top5'])`
> 
> Update progbar:
> `kbar.update(i, values=[('batch_time', batch_time.val), ('loss', losses.avg), ('top1', top1s.avg), ('top5', top5s.avg)])`
> 
> I like your work, thanks!

Oh, cool, I didn’t use stateful metrics before, and thought needs to modify the code.

Thanks!
